### PR TITLE
feat: line-height erasers exploration

### PR DESCRIPTION
**Pull request by mathisscott**
• proved out line-height eraser across multiple fonts
• ratios/measurements change for some fonts when font-weight is changed...
• ...but none of the fonts we care about
• code in demos forms foundation for the LHEs Core is delivering
• added line-height eraser demo to dev-app
• added font-autopsy demo to dev-app

Signed-off-by: Scott Mathis <smathis@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce? See bulletpoints ☝️ 

* [ ] Bugfix
* [ ] Feature
* [ ] Code style update (formatting, local variables)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] clarity.design website / infrastructure changes
* [x] Other... Please describe: adds a couple of pages to the devapp for now.

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

### CODE REVIEW

**Comment by gnomeontherun** [comment]
This is just in the dev app, I'm not clear what parts of this are just the demo and what are the eraser implementation.

**Comment by mathisscott** [comment]
> This is just in the dev app, I'm not clear what parts of this are just the demo and what are the eraser implementation.

None of this is the implementation (although the logic/code in the demos here are the same as the implementation). The implementation is in core and will be in the layouts PR that @coryrylan and I are working on.

We felt these explorations would be useful in the dev app, however, as it gave people who want to use Core with a font that isn't Metropolis a way to use the line-height erasers. It's also something we need access to for our Shape presentation so having it in the dev-app seemed like a good place to hang on to it for now...

File: src/dev/src/app/typography/typography-font-autopsy.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    return pixelifyValues(millifyValues(parseFloat(this.model.value.ascender)));
+  }
+
+  get xHeightHeight() {
+    return pixelifyValues(millifyValues(parseFloat(this.model.value.xHeight)));
+  }
+
+  get descenderOverageHeight() {
+    return pixelifyValues(millifyValues(parseFloat(this.model.value.descenderOverage)));
+  }
```
Comment (coryrylan) [inline src/dev/src/app/typography/typography-font-autopsy.ts]: Not something that needs to be fixed but more a side note, getters bound to templates are typically an anti-pattern in Angular as all the getters will fire and re-excute all the code each change detection cycle.

Comment (mathisscott) [reply src/dev/src/app/typography/typography-font-autopsy.ts]: Good to know. ❤️

**Pull request merged**